kernel/libc.c

```diff
@@ -527,6 +527,11 @@ int unlink(const char* pathname)
     return (int)myst_syscall_ret(myst_syscall_unlink(pathname));
 }
 
+int unlinkat(int dirfd, const char* pathname, int flags)
+{
+    return (int)myst_syscall_ret(unlinkat(dirfd, pathname, flags));
+}
+
```

Comment:
No need to change libc.c

Reply:
Change has been reverted. 

Context: `libc.c` is used for gcov, no additional syscall support needed.